CHANGELOG.md

```diff
@@ -289,9 +299,19 @@ for info on changes for earlier releases.
 [`no-mutable-exports`]: ./docs/rules/no-mutable-exports.md
 [`prefer-default-export`]: ./docs/rules/prefer-default-export.md
 [`no-restricted-paths`]: ./docs/rules/no-restricted-paths.md
-
+<<<<<<< HEAD
+[`no-absolute-path`]: ./docs/rules/no-absolute-path.md
+=======
+[`max-dependencies`]: ./docs/rules/max-dependencies.md
+>>>>>>> tizmagik/feature/max-dependencies
```

Comment:
Yeah, there should be pre-commit hooks that warn when there are strings like `<<<<<<<` anywhere in your project. Maybe an idea for https://github.com/SamVerschueren/clinton @SamVerschueren?